src/clientapi/index.js

```diff
@@ -118,18 +118,25 @@ exports.ClientAPI = Class(lib.PubSub, function () {
   import .UI;
   this.ui = new UI();
 
-
-  // this.track({
-  //  name: "campaignID",
-  //  category: "campaign",
-  //  subcategory: "id",
-  //  data: campaign
-  // });
-
   var map;
   try {
     if (GLOBAL.CACHE) {
       map = JSON.parse(GLOBAL.CACHE['spritesheets/map.json']);
+
+      // Add some defaults
+      for (var key in map) {
```

Comment:
they are defined by the spriter, this is just matching what the spriter currently does.  if these are not needed here, we should also remove them from the spriter